integration/use-cases/create-private-registry.js

```diff
@@ -16,31 +16,41 @@ test("Creates a private registry", async () => {
 
   const randomNumber = Math.floor(Math.random() * Math.floor(100));
   const repoName = "my-repo-" + randomNumber;
-  await page.type("cds-form-group > cds-input:nth-child(1) > input", repoName);
+  await page.type("input[placeholder=\"example\"]", repoName);
 
   await page.type(
-    "cds-form-group > cds-input:nth-child(2) > input",
+    "input[placeholder=\"https://charts.example.com/stable\"]",
     "http://chartmuseum-chartmuseum.kubeapps:8080"
   );
 
   await expect(page).toClick("label", { text: "Basic Auth" });
 
   // Credentials from e2e-test.sh
-  await page.type("cds-form-group > div:nth-child(3) > div > div:nth-child(1) > cds-input:nth-child(1) > input", "admin");
-  await page.type("cds-form-group > div:nth-child(3) > div > div:nth-child(1) > cds-input:nth-child(3) > input", "password");
+  await page.type("input[placeholder=\"Username\"]", "admin");
+  await page.type("input[placeholder=\"Password\"]", "password");
 
   // Open form to create a new secret
   const secret = "my-repo-secret" + randomNumber;
-  await expect(page).toClick(".btn-info-outline", { text: "Add new credentials" });
-  await page.type("cds-form-group > div.docker-creds-subform > cds-input:nth-child(2) > input", secret);
+  try {
+    // TODO(andresmgot): Remove this line once 2.3 is released
```

Comment:
I don't know, but couldn't we do something like: 

`await expect(page).toClick('button', { text: '"Add new credentials' })`

Or 

`await expect(page).toClick({type:'xpath', value:'<xpath to the button>'}, { text: '"Add new credentials' })`

Reply:
yes, that's what I tried but the first doesn't work because:

first option: The cds-button is the one containing the text, and then there is a hidden, empty button. Something like `<cds-button>Add new credentials<button></button><cds-button>`

second option: The path changes from one version to the other, I could not find a path common to those two.

Reply:
Ah, I see. No problem, the try-catch hotfix is ok though!